sources/packages/backend/libs/sims-db/src/entities/msfaa-number.model.ts

```diff
@@ -93,6 +93,7 @@ export class MSFAANumber extends RecordDataModel {
    * Student id related to this MSFAA.
    */
   @RelationId((msfaaNumber: MSFAANumber) => msfaaNumber.student)
+  @Column({ name: "student_id" })
```

Comment:
same in here and 111. Am i missing anything

Reply:
I believe no @ann-aot . I believe it the left over code of @sh16011993 trying few things on `@RelationId`